src/Protocol/Protocol17x.cpp

```diff
@@ -3015,6 +3015,19 @@ void cProtocol172::WriteEntityMetadata(cPacketizer & a_Pkt, const cEntity & a_En
 
 void cProtocol172::WriteMobMetadata(cPacketizer & a_Pkt, const cMonster & a_Mob)
 {
+	// Living Enitiy Metadata
+	if(a_Mob.HasCustomName())
+	{
+		a_Pkt.WriteBEUInt8(4 << 5 | 10);
```

Comment:
I could do it. Or maybe change all Hex-Values to this format to allow easier verification.
Or maybe use a macro/inline function to make it easier to understand it.